forecast-io.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
```

Comment:
This file is going to be deprecated with the switch to the DarkSky API, don't add it to development!